Scripts/CI/README_Metadata_StyleCheck/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Using Debian slim which uses minimum packages necessary; Alpine is not preferred 
+FROM alpine:3.12
+
+# MAINTAINER is depreciated
```

Comment:
deprecated?

```suggestion
# MAINTAINER is deprecated
```

Also, if deprecated, is there a newer pattern/solution to use?

Reply:
Yes. I'm glad to finally learn that. I always just read it as 'depreciated' as in 'to lose value over time' thinking that made more sense than 'to belittle.'

Reply:
I am just removing the line entirely. It was more relevant when I was modifying Ting's work.